src/_playground/Routes.js

```diff
@@ -270,59 +273,124 @@ const routes = [
-                {sectionRoutes.map(route => (
-                    <li key={route.name}>
-                        <NavLink
-                            activeClassName='frDocs-Nav__item--active'
-                            className='frDocs-Nav__item'
-                            key={route.url}
-                            to={{ pathname: route.url }}>
-                            {route.name}
-                        </NavLink>
-                    </li>
-                ))}
+    render() {
+        let sectionRoutes;
+        let freezeItems;
+        let navItems;
+        const groupedRoutes = groupArray(this.state.filteredItems, 'section');
+        //TODO: remove navItems declaration here
+        sections.sort(sortBy('sortOrder', 'name')).map(section => {
+            if (!groupedRoutes[section.name]) {
+                return;
+            }
+            // eslint-disable-next-line no-console
+            console.log('line 304', groupedRoutes[section.omitSearch]);
+
+            sectionRoutes = groupedRoutes[section.name].sort(sortBy('sortOrder', 'name'));
+            // TODO: conditional return here, navItems and freezeItems for omitSearch:true items
+            if ( sectionRoutes[0].omitSearch ) {
+                freezeItems = (
+                    <ul className='frDocs-Nav__list' key={section.name}>
+                        <li className='frDocs-Nav__headers'>{section.name}</li>
+                        {sectionRoutes.map(route => (
+                            <li key={route.name}>
+                                <NavLink
+                                    activeClassName='frDocs-Nav__item--active'
+                                    className='frDocs-Nav__item'
+                                    key={route.url}
+                                    to={{ pathname: route.url }}>
+                                    {route.name}
+                                </NavLink>
+                            </li>
+                        ))}
+                    </ul>
+                );
+            } else {
+                navItems = (
+                    <ul className='frDocs-Nav__list' key={section.name}>
+                        <li className='frDocs-Nav__headers'>{section.name}</li>
+                        {sectionRoutes.map(route => (
+                            <li key={route.name}>
+                                <NavLink
+                                    activeClassName='frDocs-Nav__item--active'
+                                    className='frDocs-Nav__item'
+                                    key={route.url}
+                                    to={{ pathname: route.url }}>
+                                    {route.name}
+                                </NavLink>
+                            </li>
+                        ))}
+                    </ul>
+                );
+            }
+        });
+        // eslint-disable-next-line no-console
+        console.log('line 326', navItems);
```

Comment:
This `console.log` should be removed.

Reply:
Definitely will be removed for merge. :)